test/authorizers/RequesterAuthorizerWithAirnode.sol.js

```diff
@@ -148,7 +154,8 @@ describe('RequesterAuthorizerWithAirnode', function () {
           );
           requesterAuthorizerWithAirnode = await requesterAuthorizerWithAirnodeFactory.deploy(
             accessControlRegistry.address,
-            requesterAuthorizerWithAirnodeAdminRoleDescription
+            requesterAuthorizerWithAirnodeAdminRoleDescription,
+            expiringMetaCallForwarder.address
```

Comment:
Maybe after 163 we could also expect that [isTrustedForwarder()](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/metatx/ERC2771Context.sol#L20-L22) is equal to this `address`